pkg/admission/customresourcevalidation/apiserver/validate_apiserver.go

```diff
@@ -41,7 +41,6 @@ func (a apiserverV1) ValidateCreate(uncastObj runtime.Object) field.ErrorList {
 	}
 
 	errs = append(errs, validation.ValidateObjectMeta(&obj.ObjectMeta, false, customresourcevalidation.RequireNameCluster, field.NewPath("metadata"))...)
-	errs = append(errs, validateAPIServerSpecCreate(obj.Spec)...)
```

Comment:
@sjenning how is this related to the cadvisor bump?

Reply:
it isn't, but https://github.com/openshift/api/pull/375 went into `openshift/api` branch `release-4.1` and after the bump, the build failed without this change